v2/operator_claims_test.go

```diff
@@ -287,7 +287,7 @@ func Test_SystemAccount(t *testing.T) {
 }
 
 func Test_AssertServerVersion(t *testing.T) {
-	operatorWithAssertServerVer := func(t *testing.T, v string) error {
+	operatorWithAssertServerVer := func(t *testing.T, v uint) error {
```

Comment:
This test is not checking that the version is something and we expect something else, just that the version string was correct. If we make the version just a number, yes, not sure of the usefulness of this test here. If I understand, the Validate() would call ParseServerVersion() that would fail if it was not simver x.y.z.

Reply:
It still asserts that the value is deserialized matches v